### ISSUE TITLE
VB Line Commit consumes Ctrl Dot

### ISSUE THREAD

**Issue by basoundr** [dotnet/roslyn#2123]
With the below code

``` VB
Module Program
    Sub Main(args As       $$A)

    End Sub
End Module
```

Pressing Ctrl Dot, LineCommit commits which results in the below code 

``` VB
Module Program
    Sub Main(args As $$A)

    End Sub
End Module
```

Now Ctrl Dot has to be issued again to show the Generate type code action

Another example is Rename

``` VB
Class Program
    Sub M(abc As Integer)
         Dim s = abc
    End Sub
End Class
```

Start an Rename Tracking session by modifying the identifier `abc` to `abC`. Press Ctrl Dot. Instead of showing the Rename Codefix, line commit hijacks Ctrl Dot to commit and the Inline rename session is lost

[dpoeschl] Adding @ljw1004 's repro video for the RenameTracking case from #3508:
![postofactofailure](https://cloud.githubusercontent.com/assets/3316258/8169510/ed275a1a-135e-11e5-9304-46b6c3d8d0ae.gif)

**Comment by Pilchie** [comment]
Hey @olegtk, @DustinCampbell was mentioning that this is likely triggered because clicking on the lightbulb triggers a `AggregateFocusLost`.  Should it?  Shouldn't we treat this more like completion?

**Comment by olegtk** [comment]
Hmmm. There is no particular reason why clicking on LightBulb should trigger AggregateFocusLost, it should be very similar to completion indeed (except that the LightBulb is hosted in the view margin). Does it also happen with QuickInfo based LightBulb you get on error squiggles?

**Comment by DustinCampbell** [comment]
FWIW, unless it was fixed in Dev14, the completion list can also result in an AggregateFocusLoss when double-clicking an item in the list to commit it. We had to specifically set focus to the editor in Roslyn in this case to keep the event from firing and VB line commit running as a result.

**Comment by CyrusNajmabadi** [comment]
Closing out as something we haven't heard anything about from customers.  Unlikely to change these vb experiences now for risk of breaking people.
